packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -143,17 +143,17 @@ export default class SelectOpener extends React.Component<
             ...sharedProps
         } = this.props;
 
-        const stateStyles = _generateStyles(isPlaceholder, error);
-
         const iconColor = disabled
             ? semanticColor.core.foreground.disabled.default
             : theme.opener.color.icon;
 
         const style = [
             styles.shared,
-            stateStyles.default,
-            disabled && stateStyles.disabled,
-            !disabled && this.state.pressed && stateStyles.press,
+            styles.default,
+            disabled && styles.disabled,
```

Comment:
For consistency, the selects also have a [read only state](https://www.figma.com/design/EuFu0U7gqc1koXm8ZhlOLp/%E2%9A%A1%EF%B8%8F-Components?node-id=9492-1546&t=3aHtZciAj58GVKsM-4) 

We can create a separate ticket for this later on if you prefer!

Reply:
I created https://khanacademy.atlassian.net/browse/WB-2046 to track that for later.